pkgs/development/lisp-modules/from-quicklisp/barebones-quicklisp-expression.sh

```diff
@@ -51,7 +51,7 @@ url="${ql_src##* }"
 [ "$ql_src_type" = github ] && {
   ql_src_type=git
   url="https://github.com/$url";
-  version="$("$(dirname "$0")/../../../build-support/upstream-updater/urls-from-page.sh" "$url/releases/" | grep /tag/ | head -n 1 | xargs -l1 basename)"
```

Comment:
It's used a few times in this file, each one needs to be updated.

Reply:
sorry forgot to check. thanks for noticing